.gitignore

```diff
@@ -10,3 +10,4 @@ target
 # Gradle files
 .gradle/*
 build/*
+bin/*
```

Comment:
OOC, what triggered this change?

Reply:
So I'm not sure -- but every time `gradelw test` I get this new `bin` folder at the root. I figured it should be added to prevent accidental addition. I'm not sure why it hasn't been added before though 🤔

Reply:
I did some googling and it looks like this happens with Eclipse. Are you using Eclipse?

Reply:
Nope. Just the command line `./gradlew test` with VSCode. I don't *think* VSCode is doing anything as I'm initiating all commands through the command line. However I am using the VSCode java plugin which may or may not be doing something. I can't seem to reproduce it anymore though! 🤔

So I guess I'll remove this now. Something made this folder but I can't seem to figure out what